src/app/period/period.component.ts

```diff
@@ -270,6 +275,22 @@ export class PeriodComponent implements OnInit {
     ).subscribe((team?: Team) => {
       if (team) {
         this.setTeam(new ImmutableTeam(team));
+        if (team.teamPermissions !== undefined) {
+          const user = this.authService.user$.getValue();
+          const userEmail = user?.email;
+          const userDomain = user?.domain;
+          const principalTypeEmail = 'email';
+          const principalTypeDomain = 'domain';
+          team.teamPermissions.write.allow.forEach(permission => {
+            if ((permission.type === principalTypeDomain && permission.id.toLowerCase() === userDomain?.toLowerCase()) ||
+              (permission.type === principalTypeEmail && permission.id.toLowerCase() === userEmail?.toLowerCase())) {
+              this.userHasEditPermissions = true;
+            }
+          });
+          if (!this.userHasEditPermissions) {
+            this.notificationService.error$.next('You do not have editing priviliges.');
```

Comment:
Why do we show an error when a user has read-only access? Opening a read-only team is perfectly normal - it's not an error.

Can't we just leave the edit button greyed out, maybe with a mouseover that says the user doesn't have permission?

Reply:
Whoops, I had this as a notification and then didn't think further when I merged important_notification and error.
I looked into making a tooltip, but creating a tooltip for a disabled element is not really supported in angular. I've left it for now.